superset/migrations/versions/d891f321997f_add_dataset_name_column_to_tables.py

```diff
@@ -0,0 +1,40 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add_dataset_name_column_to_tables
+
+Revision ID: d891f321997f
+Revises: 19e978e1b9c3
+Create Date: 2021-04-29 16:34:15.679026
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "d891f321997f"
+down_revision = "19e978e1b9c3"
+
+import sqlalchemy as sa
+from alembic import op
+
+
+def upgrade():
+    op.add_column(
+        "tables", sa.Column("dataset_name", sa.String(length=250), nullable=True)
+    )
+
+
+def downgrade():
+    op.drop_column("tables", "dataset_name")
```

Comment:
Sqlite does not support dropping column.
Don't know any way around.

Reply:
You can use this: https://alembic.sqlalchemy.org/en/latest/batch.html